src/shared/btrfs-util.c

```diff
@@ -1637,13 +1637,10 @@ int btrfs_qgroup_find_parents(int fd, uint64_t qgroupid, uint64_t **ret) {
                         break;
         }
 
-        if (n_items <= 0) {
-                *ret = NULL;
-                return 0;
-        }
+        assert((n_items > 0) == (items != NULL));
```

Comment:
we usually don't explicit NULL comparisons?

hence turn rhs into `!!items`?

Reply:
Updated like that.